views/channel.ecr

```diff
@@ -7,71 +7,76 @@
   </head>
   <body>
     <% render "partials/header" %>
-    <main>
-      <section class="card cols-6">
-        <h3>Details</h3>
-        <table class="details-table">
-          <tr>
-            <th>Connection</th>
-            <td id="ch-connection"></td>
-          </tr>
-          <tr>
-            <th>Username</th>
-            <td id="ch-username"></td>
-          </tr>
-          <tr>
-            <th>State</th>
-            <td id="ch-state"></td>
-          </tr>
-          <tr>
-            <th>Mode</th>
-            <td id="ch-mode"></td>
-          </tr>
-        </table>
-      </section>
-      <section class="card cols-6">
-        <h3>Stats</h3>
-        <table class="details-table">
-          <tr>
-            <th>Messages unacknowledged</th>
-            <td id="ch-unacked"></td>
-          </tr>
-          <tr>
-            <th>Prefetch count</th>
-            <td id="ch-prefetch"></td>
-          </tr>
-          <tr>
-            <th>Global prefetch count</th>
-            <td id="ch-global-prefetch"></td>
-          </tr>
-        </table>
-      </section>
-      <section class="card">
-        <h3>Message rates</h3>
-        <div class="chart-container" id="chart"></div>
-      </section>
-      <section class="card">
-        <h3>
-          Consumers
-          <small id="consumer-count"></small>
-        </h3>
-        <div class="table-wrapper">
-          <div id="table-error"></div>
-          <table id="table" class="table">
-            <thead>
-              <tr>
-                <th>Consumer tag</th>
-                <th class="left">Queue</th>
-                <th>Ack required</th>
-                <th>Exclusive</th>
-                <th>Prefetch count</th>
-              </tr>
-            </thead>
-            <tbody></tbody>
-          </table>
+    <div class="grid-content">
+      <main>
+        <div id="breadcrumbs" class="cols-12">
+            <h2 class="page-title"><%=pagename%><span class="breadcrumb-divider">/</span> <div id="pagename-label"></div></h2>
```

Comment:
that `breadcrumb-divider` span can be `::after` thing instead?

Reply:
Great point. Now fixed.